src/devtools.h

```diff
@@ -120,4 +120,6 @@ struct devtool_node_status {
  */
 extern void devtool_get_node_status(meshlink_handle_t *mesh, meshlink_node_t *node, devtool_node_status_t *status);
 
+extern meshlink_submesh_t **devtool_get_all_submeshes(meshlink_handle_t *mesh, meshlink_submesh_t **submeshes, size_t *nmemb);
```

Comment:
You also need to write the Doxygen documentation for this function, see the documentation of `devtool_get_all_edges()` in this same header file.

Reply:
I added the documentation for that function